src/status_im2/navigation/roots.cljs

```diff
@@ -136,6 +137,16 @@
                                     (status-bar-options)
                                     {:topBar (assoc (topbar-options) :visible false)})}}}})
 
+;; Theme Order for navigation roots
+;; 1. Themes hardcoded in below map
```

Comment:
more of a question, then a comment. The designs for syncing are also only in dark mode. Should I make them a stack or should we configure that we can override the theme on the screen configuration? 🤔 

Designs for reference:
https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?node-id=1-19&t=c8o2zYIXDezF6A3V-0

Reply:
Hi @J-Son89, Thank you very much for reviewing PR.
From designs it look like profile is also in dark-mode. So maybe design team only designed for dark mode now, and screen will need to support both themes. Not sure
cc: @pedro-et

Reply:
@Parveshdhull, My question was not about the designs. I have already discussed with @pedro-et and Syncing is a dark theme.
That said, I am asking how we best handle this in code and if you have any suggestions.

Reply:
e.g we make syncing it's own stack in navigation... or in screens.cljs we can add something like 

```
    {:name      :settings-syncing
     :theme    :dark  (perhaps adding an option at the route level??)
     :insets    {:bottom true}
     :options   {:topBar {:title {:text (i18n/label :t/syncing)}}}
     :component settings-syncing/views}
```

Reply:
we need to use screens.cljs, also why we even need to use :theme :dark here?
If these screens are only going to store dark theme, we can avoid using functions like
`colors/theme-color` and just hard code a single value

Reply:
Only thing we might need to do manually is changing staus-bar-color while navigating.
But I didn't knew if profile setting will only be in dark mode, if this is the case then you don't need to change status bar color, it will be already changed when opening profile settings

Reply:
That's also true, it's just a lot more convenient when using the quo2 components as then we do not need to configure each prop

Reply:
Better to explore more when implementing profiles settings screen, but we already have case like this which only supports single theme - shell/jump-to screen